store/src/java/com/zimbra/cs/service/admin/ToXML.java

```diff
@@ -66,6 +73,15 @@ public static Element encodeAccount(Element parent, Account account,
             }
         }
         Map attrs = account.getUnicodeAttrs(applyCos);
+        if (isEffectiveQuota) {
+            try {
+                // setting effective quota value refer ZBUG-1869
+                long effectiveQuota = AccountUtil.getEffectiveQuota(account);
+                attrs.put(Provisioning.A_zimbraMailQuota, Long.toString(effectiveQuota));
```

Comment:
In GetInfo.java, `String.valueOf` is used. Is there any reason why `Long.toString` is used here?
I think `String.valueOf` is better to avoid NullPointerException.

Reply:
Purpose of both is same, yes agreed, String.valueOf avoid NullPointer . Now using String.valueOf()